release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,21 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-4-17-nw-sriov-day1-nics"]
+==== NIC partitioning for SR-IOV devices general availability (GA)
+
+With this update, the ability to enable NIC partitioning for Single Root I/O Virtualization (SR-IOV) devices at install time is moving from Technology Preview support to GA.
+
+For more information, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
```

Comment:
The page linked to still has TP in the title (reminder just in case).

Reply:
Thanks for the reminder. That is being addressed in https://github.com/openshift/openshift-docs/pull/81406 (I think you are reviewing this one also)